tests/topotests/ospfapi/test_ospf_clientapi.py

```diff
@@ -1684,7 +1694,9 @@ def _test_opaque_link_local_lsa_crash(tgen, apibin):
                             "linkStateId": "230.0.0.1",
                             "advertisingRouter": "1.0.0.0",
                             "lsaSeqNumber": "80000001",
-                            "opaqueData": "feedaceecafebeef",
+                            "opaqueData": {
+                              "opaqueData": "feedaceecafebeef",
```

Comment:
Just noticed, that this is a new commit since the last review. Is this supposed to be double-named? I mean opaqueData->opaqueData? Are there more fields under the first opaqueData?

Reply:
Yes and no. I use the generic key "opaqueData" to enclose all TLV carried by the Opaque LSA. Then, I discover that ospf client api uses also the generic term opaqueData.

To avoid this double name, let me know what do you prefer:
-  uses "opaqueTLV" as top key for the Opaque Data
- replace the "opaqueData" key by the simple "data" key in ospf api client

Reply:
To me personally the first sounds better because it clearly explains what's behind it. And keeping just `data` is too generic. What others think?

Reply:
I like the first better, as well ...

Reply:
I finally change for TLV but a little bit different from what I originally proposed. There is already `opaqueType` and `opaqueDataLenght`. Thus, I change for `opaqueValues` (the V of TLV) and change `opaqueDataLength` by `opaqueLength`. Thus, we got the Type, Length and Values of the TLV encoding of Opaque LSA. I hope it is fine for you.